spec/go-spec.coffee

```diff
@@ -299,81 +299,58 @@ describe 'Go grammar', ->
       funcVal = funcVals[funcs.indexOf(func)]
       {tokens} = grammar.tokenizeLine func
       expect(tokens[0].value).toEqual funcVal
-      expect(tokens[0].scopes).toEqual ['source.go', 'support.function.built-in.go']
+      expect(tokens[0].scopes).toEqual ['source.go', 'support.function.builtin.go']
 
   it 'tokenizes operators', ->
-    opers = [
-      '+', '&', '+=', '&=', '&&', '==', '!=', '-', '|', '-=', '|=', '||', '<',
-      '<=', '*', '^', '*=', '^=', '<-', '>', '>=', '/', '<<', '/=',
-      '<<=', '++', '=', ':=', '%', '>>', '%=', '>>=', '--', '!', '...',
-      '&^', '&^='
-    ]
-
-    for op in opers
-      {tokens} = grammar.tokenizeLine op
-
-      fullOp = tokens.map((tok) -> tok.value).join('')
-      expect(fullOp).toEqual op
-
-      scopes = tokens.map (tok) -> tok.scopes
-      allKeywords = scopes.every (scope) -> 'keyword.operator.go' in scope
-
-      expect(allKeywords).toBe true
-
-  it 'tokenizes bracket operators', ->
-    opers = [
-      {
-        values: [ '[', ']' ]
-        scope: 'meta.brace.square.go'
-      }
-      {
-        values: [ '(', ')' ]
-        scope: 'meta.brace.round.go'
-      }
-      {
-        values: [ '{', '}' ]
-        scope: 'meta.brace.curly.go'
-      }
-    ]
-
-    for ops in opers
+    binaryOpers =
+      'keyword.operator.arithmetic.go': ['+', '-', '*', '/', '%', '&', '|', '^', '&^', '<<', '>>']
+      'keyword.operator.assignment.go': ['=', '+=', '-=', '|=', '^=', '*=', '/=', ':=', '%=', '<<=', '>>=', '&=', '&^=']
+      'keyword.operator.channel.go': ['<-']
+      'keyword.operator.comparison.go': ['==', '!=', '<', '<=', '>', '>=']
+      'keyword.operator.decrement.go': ['--']
+      'keyword.operator.ellipsis.go': ['...']
+      'keyword.operator.increment.go': ['++']
+      'keyword.operator.logical.go': ['&&', '||']
+
+    unaryOpers =
+      'keyword.operator.address.go': ['*var', '&var']
+      'keyword.operator.arithmetic.go': ['+var', '-var', '^var']
+      'keyword.operator.logical.go': ['!var']
```

Comment:
`var` should be 'keyword.variable.go' Otherwise styles for `keyword.operator` can affect on `var` symbol